lib/FrontendTool/FrontendTool.cpp

```diff
@@ -747,6 +747,21 @@ static void emitReferenceDependencies(CompilerInvocation &Invocation,
   }
 }
 
+static bool
+finishCompilationWhenOnlyTypechecking(CompilerInvocation &Invocation,
```

Comment:
Since both of these actions also happen in other compilation modes (just later), it seems like this is the wrong name. We ought to be able to use the function in both places.

(…if it's worth it at all, since these two actions aren't inherently related.)

Reply:
Fair point, I guess I'll not try to factor that.